package-lock.json

```diff
@@ -0,0 +1,6961 @@
+{
```

Comment:
Why do we have so many changes to this file?

Reply:
Good catch, this was added by mistake, deleted!